src/cmd-diff

```diff
@@ -291,14 +310,14 @@ DIFFERS = [
            needs_ostree=False, function=diff_live_initrd_tree),
     Differ("live-initrd", "Diff live initramfs content",
            needs_ostree=False, function=diff_live_initrd),
-    Differ("live-rootfs-ls", "Diff live rootfs listings",
-           needs_ostree=False, function=diff_live_rootfs_tree),
-    Differ("live-rootfs", "Diff live rootfs content",
-           needs_ostree=False, function=diff_live_rootfs),
-    Differ("live-squashfs-ls", "Diff live squashfs listings",
-           needs_ostree=False, function=diff_live_squashfs_tree),
-    Differ("live-squashfs", "Diff live squashfs content",
-           needs_ostree=False, function=diff_live_squashfs),
+    Differ("live-rootfs-img-ls", "Diff live-rootfs.img listings",
+           needs_ostree=False, function=diff_live_rootfs_img_tree),
+    Differ("live-rootfs-img", "Diff live-rootfs.img content",
+           needs_ostree=False, function=diff_live_rootfs_img),
+    Differ("live-sysroot-ls", "Diff live '/root.[erofs|squash]fs' (embed into live-rootfs) listings",
+           needs_ostree=False, function=diff_live_sysroot_tree),
+    Differ("live-sysroot", "Diff live '/root.[ero|squash]fs' (embed into live-rootfs) content",
+           needs_ostree=False, function=diff_live_sysroot),
```

Comment:
I think these could be named better, but I'll argue that point in a separate PR and see what other people think.

functionally this should work great!

Reply:
I'll hold off reviewing this hunk until your PR. :)